.github/workflows/prod-environment.yml

```diff
@@ -206,7 +206,7 @@ jobs:
         run: |
           source .venv/bin/activate
           source nrlf.sh
-          nrlf test smoke
+          nrlf test smoke producer ${{ inputs.environment }}
```

Comment:
we don't want ${{ inputs.environment }} here do we?

Reply:
yeah you still need to provide the environment for the command so it can get the right oauth details from secrets manager